File: cinema-backend/src/main/java/org/example/cinemabackend/user/application/dto/request/UpdateCinemaManagerRequest.java
package org.example.cinemabackend.user.application.dto.request;

import jakarta.validation.constraints.Email;
import jakarta.validation.constraints.NotBlank;
import jakarta.validation.constraints.Size;

public record UpdateCinemaManagerRequest(
        @NotBlank @Size(max = 255) String firstName,
        @NotBlank @Size(max = 255) String lastName,
        @NotBlank @Email @Size(max = 255) String email,
        String managedCinemaName
) {
}

File: cinema-frontend/src/app/user/dtos/request/update-cinema-manager.request.ts
export class UpdateCinemaManagerRequest {
  constructor(
    public readonly firstName: string,
    public readonly lastName: string,
    public readonly email: string,
    public readonly currentPassword: string,
    public readonly newPassword: string,
    public readonly managedCinemaName: string | null,
  ) {
  }
}
